src/Components/Facility/HospitalList.tsx

```diff
@@ -37,11 +41,17 @@ export const HospitalList = () => {
 
   useAbortableEffect(
     (status: statusType) => {
+      handleDownload();
```

Comment:
Codacy found an issue: ['handleDownload' was used before it was defined.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5807708)